shared/actions/chat2/index.js

```diff
@@ -2005,7 +2005,7 @@ function* chat2Saga(): Saga.SagaGenerator<any, any> {
   // Platform specific actions
   if (isMobile) {
     // Push us into the conversation
-    yield Saga.safeTakeEveryPure(Chat2Gen.selectConversation, mobileNavigateToThread)
+    yield Saga.safeTakeEveryPure(Chat2Gen.selectConversation, mobileNavigateOnSelect)
```

Comment:
Why do we do this only for mobile?

Reply:
on mobile selecting a conversation changes your navigation on desktop the routing doesn't change at all